src/stateToHTML.js

```diff
@@ -344,26 +344,29 @@ class MarkupGenerator {
         }
         return content;
       }).join('');
+
       let entity = entityKey ? Entity.get(entityKey) : null;
       // Note: The `toUpperCase` below is for compatability with some libraries that use lower-case for image blocks.
       let entityType = (entity == null) ? null : entity.getType().toUpperCase();
-      let { entityStyles } = this.options;
       if (entityType != null && entityType === ENTITY_TYPE.LINK) {
-        let attrs = DATA_TO_ATTR.hasOwnProperty(entityType) ? DATA_TO_ATTR[entityType](entityType, entity) : null;
-        let additionalAttrs = entityStyles && entityStyles[entityType] && entityStyles[entityType].attributes || {}
-        let attrString = stringifyAttrs({ ...attrs, ...normalizeAttributes(additionalAttrs) });
+        let attrString = this.getAttributesWithEntityStyles(entityType, entity);
         return `<a${attrString}>${content}</a>`;
       } else if (entityType != null && entityType === ENTITY_TYPE.IMAGE) {
-        let attrs = DATA_TO_ATTR.hasOwnProperty(entityType) ? DATA_TO_ATTR[entityType](entityType, entity) : null;
-        let additionalAttrs = entityStyles && entityStyles[entityType] && entityStyles[entityType].attributes || {}
-        let attrString = stringifyAttrs({ ...attrs, ...normalizeAttributes(additionalAttrs) });
+        let attrString = this.getAttributesWithEntityStyles(entityType, entity);
         return `<img${attrString}/>`;
       } else {
         return content;
       }
     }).join('');
   }
 
+  getAttributesWithEntityStyles(entityType: string, entity: Entity): string {
+    let { entityStyles } = this.options;
```

Comment:
You don't need to use "let" everywhere.. if it doesn't have to be mutable, use constants instead. *Slowly walks away backwards*

Reply:
Admittedly, this is from my style of coding. It's evident all over this repo and others. I use let by default unless I need an explicit constant such as `const DEFAULT_WIDTH = 20` or something like that. Maybe it's just habitual on my part because I see the JS community moving more towards using const by default and let only when necessary.